joomla/joomla-cms
# [4.0][RFC] npm build tools part1 - templates

### Summary of Changes
Since https://github.com/joomla/joomla-cms/pull/21294 raised way too many questions and it's pretty obvious that people couldn't follow what was happening in there, here is a PR with less files touched.
So this PR is moving the assets of the templates to the source folder
- naming and placement is debatable, eg /build/sources or /media-source or whatever
- right now the command `node build.js --compile-css` will get us where we were before the PR. The problem is that to do so we're hardcoding the paths and that is why I proposed that the css files of the templates should be on a folder in the media folder (an exact copy of the media-source structure needs 0 hardcoding)

### Testing Instructions
Fetch, pull, run `node build.js --compile-css` and check that templates (the css part, js is not covered here) is still intact
